lib/k8s/client.ex

```diff
@@ -98,7 +98,7 @@ defmodule K8s.Client do
   def apply(resource, mgmt_params \\ []) do
     field_manager = Keyword.get(mgmt_params, :field_manager, @mgmt_param_defaults[:field_manager])
     force = Keyword.get(mgmt_params, :force, @mgmt_param_defaults[:force])
-    Operation.build(:apply, resource, field_manager: field_manager, force: force)
```

Comment:
I would also use `:patch` now for apply.

```suggestion
    Operation.build(:patch, api_version, kind, path_params, subresource,
```

Reply:
Done.  I assume you want to only change apply to patch, and keep the field manager and force?

Reply:
Yes exactly.